src/services/local-storage-double.js

```diff
@@ -0,0 +1,9 @@
+export class FakeLocalStorage {
```

Comment:
I would use a simpler version in `experimental-features.test.js`, but this file looks nice and useful in other scenarios.

Maybe we should move it to a specific folder like `/test-utils` or something like that.